src/main.cpp

```diff
@@ -63,9 +76,15 @@ void loop() {
             auto written = current_client.write(buffer, available);
             Serial.peekConsume(written);
             lastUpdate = now;
+            if (written > 0)
+                digitalWrite(LED_BUILTIN, !HIGH);
         }
         else {
-            Serial.peekConsume(0);
+            Serial.peekConsume(available);
```

Comment:
why do we switch to `peekConsume(available)`? this now drops everything that we haven't forwarded?